app/routes/overview/components/Overview.js

```diff
@@ -13,59 +13,59 @@ import Feed from './Feed';
 import CompactEvents from './CompactEvents';
 import { EVENT_TYPE_TO_STRING, colorForEvent } from 'app/routes/events/utils';
 import Button from 'app/components/Button';
-import type { Event } from 'app/models';
 
 const TITLE_MAX_LENGTH = 50;
 const DESCRIPTION_MAX_LENGTH = 140;
 const IMAGE_HEIGHT = 192;
 
-function PrimaryItem({ event }: { event: ?Event }) {
-  if (!event) {
+function PrimaryItem({ item }: { item: ?Object }) {
```

Comment:
Better to use a flow type that accepts Event or Article. `Event | Article` instead of Object.